contentctl/actions/detection_testing/views/DetectionTestingView.py

```diff
@@ -143,33 +171,43 @@ def getSummaryObject(
         # Compute total detections
         total_detections = total_fail + total_pass + total_untested + total_skipped
 
+        # Compute total detections actually tested (at least one test not skipped)
+        total_tested_detections = total_fail + total_pass
 
         # Compute the percentage of completion for testing, as well as the success rate
         percent_complete = Utils.getPercent(
             len(tested_detections), len(untested_detections), 1
         )
         success_rate = Utils.getPercent(
-            total_pass, total_detections-total_skipped, 1
+            total_pass, total_tested_detections, 1
         )
 
-        # TODO (#230): expand testing metrics reported
+        # TODO (cmcginley): the dual language of skipped_detections and untested_detections is a
+        #   bit confusing; consider renaming
+        # TODO (#230): expand testing metrics reported (and make nested)
         # Construct and return the larger results dict
         result_dict = {
+            # TODO (cmcginley): differentiate total detections vs total tested
             "summary": {
+                "mode": self.config.getModeName(),
+                "enable_integration_testing": self.config.enable_integration_testing,
                 "success": overall_success,
                 "total_detections": total_detections,
+                "total_tested_detections": total_tested_detections,
                 "total_pass": total_pass,
                 "total_fail": total_fail,
                 "total_skipped": total_skipped,
                 "total_untested": total_untested,
-                "total_experimental_or_deprecated": len(deprecated_detections+experimental_detections),
+                "total_production": total_production,
+                "total_experimental": total_experimental,
+                "total_deprecated": total_deprecated,
+                "total_manual": total_manual,
+                "total_other_skips": total_skipped - total_deprecated - total_experimental - total_manual,
                 "success_rate": success_rate,
             },
             "tested_detections": tested_detections,
+            "skipped_detections": skipped_detections,
             "untested_detections": untested_detections,
             "percent_complete": percent_complete,
-            "deprecated_detections": deprecated_detections,
```

Comment:
How do we feel about getting rid of these lists? They don't seem to do much for us, and now these detections will show up with everything else we skip under `skipped_detections`

Reply:
Given that there is now some additional logic:
```
# Skip tests for non-production detections
        if self.status != DetectionStatus.production.value:                                         # type: ignore
            self.skip_all_tests(f"TEST SKIPPED: Detection is non-production ({self.status})")
``` 
in the detection_abstract.py class, I think this makes sense.